cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -140,7 +142,40 @@ def test_validate_not_required_dialog_element():
     Bugzilla:
         1692736
     """
-    pass
+    catalog_item, sd, _ = generic_catalog_item_with_imported_dialog
+
+    service_catalog = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
+    view = navigate_to(service_catalog, "Order")
+
+    # inputs on page
+    required = view.fields("required").input
+    validated = view.fields("validate").input
+
+    def clean_inputs():
+        required.fill("")
+        validated.fill("")
```

Comment:
Do these `fill` methods return `True`? If so,
```suggestion
    def clean_inputs():
        return required.fill("") and validated.fill("")
    assert clean_inputs()
```

Reply:
@john-dupuy I created simple method as before start check better to clean all input so it will not effect on checks. if one of input is already clean then it will return False; which i don't want. I put this to just clean input not for assertion.